docs/fundamentals/systems/core/SlotMixin.md

```diff
@@ -26,13 +26,16 @@ The help text here needs to be connected to the input element that may live in s
 
 In the case above, all inputs need to be able to refer the error message of their parent.
 
-In a nutshell: SlotMixin helps you with everything related to rendering light dom (i.e. rendering to slots).
-So that you can build accessible ui components with ease, while delegating all edge cases to SlotMixin.
-Edge cases that it solves:
+**In a nutshell**: SlotMixin helps you with everything related to rendering light dom (i.e. rendering to slots).
+You can build accessible ui components with ease, while delegating all edge cases to SlotMixin.
 
-- rendering light dom in context of scoped customElementRegistries: we respect the customElementRegistry bound to your ShadowRoot
-- the concept of rerendering based on property effects
-- easily render lit templates
+Edge cases that SlotMixin solves:
+
+- **mediates** between light dom provided by the user ('public slots') and light dom provided by the component author ('private slots').
+- allows to hook into the reactive update loop of LitElement (**rerendering** on property changes)
+- respects the **scoped registry** belonging to the shadow root.
+
+## The api
 
 So, what does the api look like? SlotMixin can be used like this:
```

Comment:
An example where `AccessibleControl` is used "in action" ?

Reply:
It's a fictional control that should indicate that SlotMixin contributes to a11y. We could emphasize this by calling it `MyAccessibleControl` if you want